packages/flutter/test/widgets/widget_inspector_test.dart

```diff
@@ -394,16 +395,19 @@ class _TestWidgetInspectorService extends TestWidgetInspectorService {
         ),
       );
 
-      expect(getInspectorState().selection.current, isNull); // ignore: avoid_dynamic_calls
+      expect(WidgetInspectorService.instance.selection.current, isNull); // ignore: avoid_dynamic_calls
```

Comment:
Can we remove the ` // ignore:` here now? This should no longer be a dynamic call.